src/main/kotlin/test/ktortemplate/core/utils/json/JsonSettings.kt

```diff
@@ -15,6 +19,9 @@ object JsonSettings {
         coerceInputValues = false
         classDiscriminator = "type"
         allowSpecialFloatingPointValues = false
+        serializersModule = SerializersModule {
+            contextual(PageResponse.serializer(Car.serializer()))
```

Comment:
@monxalo this forces us to update the core json settings with the specific types we want to use with the PageResponse (or any other generic class)?

Reply:
Not necessarily, `SerializersModule` has a `plus()` which can add more modules. This can be setup using dependency injection where each module can provide a `SerializersModule` that contributes to `serializersModule`.

Reply:
Nevermind, these serializers are unique, we can't register another like: `PageResponse.serializer(Parts.serializer))`. Will revert this.